sqlmesh/core/engine_adapter/spark.py

```diff
@@ -265,6 +267,10 @@ def _create_table_properties(
                     this=exp.Schema(expressions=partitioned_by),
                 )
             )
+
+        for key, value in (table_properties or {}).items():
+            properties.append(exp.Property(this=key, value=exp.convert(value)))
```

Comment:
remove the convert here and do it in the validator?

Reply:
udpated